include/parakeet/Driver.h

```diff
@@ -112,80 +148,39 @@ class Driver
             unsigned char intensity[1000];
         };
 
-        struct CommandData
+        struct MessageData
         {
             int len;
             unsigned char body[8 * 1024];
```

Comment:
8*1024 is 8kb right? Is there are reason you put the calculation in there vs just the final number? Is there are reason we chose 8kb vs some other random number?

Reply:
The CommandData struct came from the original source code used to communicate with the sensor. The multiplication and array of size 8192 is just a left-over from that. What do you suggest we use instead?